tests/sentry/integrations/msteams/notifications/test_deploy.py

```diff
@@ -14,6 +15,7 @@
 )
 @patch("sentry.integrations.msteams.MsTeamsAbstractClient.send_card")
 class MSTeamsDeployNotificationTest(MSTeamsActivityNotificationTest):
+    @skip("Flaky test")
```

Comment:
please tag this with a ticket number such that it will get tracked and fixed -- you can follow the workflow in https://github.com/getsentry/rfcs/pull/27

Reply:
We didn't create a ticket for this but routed it to the correct team because we didn't know who was responsible for fixing this as git blame points to people that are no longer in the company 
@brianthi I think created a ticket for this

Reply:
please add that to the skip message -- thanks!